_Week8/Group3/README.md

```diff
@@ -0,0 +1,74 @@
+# WEEK 8 HW - Group 3
+
+## Motivation
+
+For this week's simulation assignment, we chose to illustrate the importance of intracluster correlation and the resulting design effect in clustered research designs.
+
+### ICC
+
+The intracluster correlation coefficient is a measure of how data point within clusters are related. The ICC is an important consideration when conducting power calculations and interpreting the results of data from a clustered study design. This is because similarity among the observations in clusters makes it more difficult to detect the true differences between study arms by reducing response variability. The ICC measures this similarity by comparing within cluster variance to between cluster variance.
+
+Note that accounting for similarities among clustered subjects usually results in power loss. Thus, with higher levels of intracluster correlation, more observations are required to obtain the same minimal detectable effect.
+
+A high icc ($\rho$) represents comparatively high "between clusters" variance.
```

Comment:
Testing here, I can't get this to render correctly either -- the parenthesis seem to break the math formatting! It only works with spacing, ie, ( $\rho$ ), which is weird. Looks like it has to be written as $(\rho)$ -- with the parentheses inside the `$` -- in order to work. Good to learn!

Reply:
Thanks for the tip--I've somehow never encountered this before!